cmd/cmd_utils.go

```
@@ -665,6 +673,36 @@
+			log.Debug("git check failed", "error", err)
+		}
+		return false
+	}
+
+	return true
+}
+
+// verifyInsideGitRepo checks if we're in a git repo.
+func verifyInsideGitRepo() bool {
+	// Skip check if either env var is set
+	if os.Getenv("ATMOS_BASE_PATH") != "" || os.Getenv("ATMOS_CLI_CONFIG_PATH") != "" {
```

golangci-lint (1.64.8)

[error] use of `os.Getenv` forbidden because "Use `viper.BindEnv` for new environment variables instead of `os.Getenv`"

(forbidigo)